app/controllers/api/issues_controller.rb

```diff
@@ -19,14 +19,16 @@ def update_issue
 
     def close_issue
       user, repo, number = params[:user], params[:repo], params[:number]
-      @issue = huboard.board(user, repo).issue(number).close
-      render json: @issue
+      @issue = huboard.board(user, repo).issue(number)
+      @issue.close
+      render json: @issue.activities
```

Comment:
I decided against creating the object client-side because when we publish the event it would be nice for real time to update other users activities also - I suppose the incoming events could also create an in-memory activity object as well... thoughts ?